api/app/Repositories/EventType/EventTypeRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repositories\EventType;
+
+use App\Entity\EventType;
+use App\Repositories\BaseRepository;
+use Illuminate\Support\Collection;
+
+final class EventTypeRepository extends BaseRepository
+{
+    public function getById(int $id): EventType
+    {
+        return EventType::findOrFail($id);
```

Comment:
Use UserRepository as reference. The method should return `null` if the record was not found, so simply just use `find` method and change return type.